src/pymor/reductors/era.py

```diff
@@ -0,0 +1,160 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+import numpy as np
+import scipy.linalg as spla
+
+from pymor.algorithms.projection import project
+from pymor.algorithms.to_matrix import to_matrix
+from pymor.core.cache import cached, CacheableObject
+from pymor.models.iosys import LTIModel
+from pymor.operators.numpy import NumpyHankelOperator, NumpyMatrixOperator
+
+
+class ERAReductor(CacheableObject):
+    r"""Eigensystem Realization Algorithm reductor.
+
+    Constructs a (reduced) realization that matches the Markov parameters optimally for given order
```

Comment:
Optimal in what sense?

Reply:
I removed the word optimal and added a docstring to `error_bounds`.